cfgov/v1/models/enforcement_action_page.py

```diff
@@ -0,0 +1,215 @@
+        related_name='enforcement_dispositions'
+    )
+
+
+# Will exist until can be sourced from enforce db
+class EnforcementActionStatus(models.Model):
+    status = models.CharField(max_length=50, choices=enforcement_statuses)
+    action = ParentalKey('v1.EnforcementActionPage',
+                         on_delete=models.CASCADE,
+                         related_name='statuses')
+
+
+# Will exist until can be sourced from enforce db
+class EnforcementActionDocket(models.Model):
+    docket_number = models.CharField(max_length=50)
+    action = ParentalKey('v1.EnforcementActionPage',
+                         on_delete=models.CASCADE,
+                         related_name='docket_numbers')
+
+
+class EnforcementActionDefendantType(models.Model):
+    defendant_type = models.CharField(
+        max_length=15, choices=enforcement_defendant_types, blank=True
+    )
+    action = ParentalKey('v1.EnforcementActionPage',
+                         on_delete=models.CASCADE,
+                         related_name='defendant_types')
+
+
+class EnforcementActionProduct(models.Model):
+    product = models.CharField(
+        max_length=50, choices=enforcement_products
+    )
+    action = ParentalKey('v1.EnforcementActionPage',
+                         on_delete=models.CASCADE,
+                         related_name='products')
+
+
+class EnforcementActionAtRisk(models.Model):
+    at_risk_group = models.CharField(
+        max_length=30, choices=enforcement_at_risk_groups
+    )
+    action = ParentalKey('v1.EnforcementActionPage',
+                         on_delete=models.CASCADE,
+                         related_name='at_risk_groups')
+
+
+class EnforcementActionStatute(models.Model):
+    statute = models.CharField(
+        max_length=30, choices=enforcement_statutes
+    )
+    action = ParentalKey('v1.EnforcementActionPage',
+                         on_delete=models.CASCADE,
+                         related_name='statutes')
+
+
+class EnforcementActionPage(AbstractFilterPage):
+    public_enforcement_action = models.CharField(max_length=150, blank=True)
+    initial_filing_date = models.DateField(null=True, blank=True)
+    settled_or_contested_at_filing = models.CharField(
+        max_length=10,
+        choices=[('Settled', 'Settled'), ('Contested', 'Contested')],
+        blank=True
+    )
+    court = models.CharField(default='', max_length=150, blank=True)
+
+    content = StreamField([
+        ('full_width_text', organisms.FullWidthText()),
+        ('expandable', organisms.Expandable()),
+        ('expandable_group', organisms.ExpandableGroup()),
+        ('notification', molecules.Notification()),
+        ('table_block', organisms.AtomicTableBlock(
+            table_options={'renderer': 'html'})),
+        ('feedback', v1_blocks.Feedback()),
+    ], blank=True)
+
+    content_panels = [
+        StreamFieldPanel('header'),
+        StreamFieldPanel('content')
+    ]
+
+    metadata_panels = [
```

Comment:
Well I think what we should be matching these fields to is the internal template which itself is informed from the (currently staged) "About the Data" page. We'll be updating the template so that the about the data page column order + template order + wagtail field order are all the same, which I think is the more important thing for content editors here.